content/de/docs/concepts/architecture/cloud-controller.md

```diff
@@ -0,0 +1,238 @@
+---
+title: Concepts Underlying the Cloud Controller Manager
+content_template: templates/concept
+weight: 30
+---
+
+{{% capture overview %}}
+Das Konzept des Cloud Controller Managers (CCM) (nicht zu verwechseln mit der Binärdatei) wurde ursprünglich entwickelt, um Cloud-spezifischen Anbieter Code und den Kubernetes Kern unabhängig voneinander entwickeln zu können. Der Cloud Controller Manager läuft zusammen mit anderen Master Komponenten wie dem Kubernetes Controller Manager, dem API-Server und dem Scheduler auf dem Host. Es kann auch als Kubernetes Addon gestartet werden, in diesem Fall läuft er auf Kubernetes.
+
+Das Design des Cloud Controller Managers basiert auf einem Plugin Mechanismus, der es neuen Cloud Anbietern ermöglicht, sich mit Kubernetes einfach über Plugins zu integrieren. Es gibt Pläne für die Einbindung neuer Cloud Anbieter auf Kubernetes und für die Migration von Cloud Anbietern vom alten Modell auf das neue CCM-Modell.
+
+Dieses Dokument beschreibt die Konzepte hinter dem Cloud Controller Manager und gibt Details zu den damit verbundenen Funktionen.
+
+Die Architektur eines Kubernetes Clusters ohne den Cloud Controller Manager sieht wie folgt aus:
+
+![Pre CCM Kube Arch](/images/docs/pre-ccm-arch.png)
```

Comment:
Nicht übersetzt: “Pre CCM Kube Arch”

Reply:
Zu "CCM Kube Arch" siehe unten. "Pre" muss nicht zwangsläufig übersetzt werden, in diesem Kontext habe ich "Pre" als Anglizismus betrachtet, darüber hinaus behält es so die Konsistenz zum Datei Namen.